README.md

```diff
@@ -35,7 +31,50 @@ docker compose exec driver rake validate_all
 
 Output from the command will appear in your terminal.
 
-## Development Mode
+## Terminology
+
+The production UK federation tooling performs *checks* on metadata.
+
+The purpose of this testbed is to perform *tests* on those checks.
+
+Each test is composed of an XML *test file* and an optional
+sidecar *options file* controlling the execution of the test.
+
+A test is executed by sending the *test data* from the test file to
+one or more *endpoints* providing a metadata validation web service.
+
+Each addressed web service runs a requested *validator* on the test data.
+This is a named sequence of MDA stages; the result is a possibly empty
+set of *status metadata* which is returned to the testbed driver.
+
+A test *succeeds* if the returned status metadata matches the
+test's expected results. The test *fails* if any differences
+are detected.
+
+## Endpoints
+
+The `docker compose` deployment currently includes four containers
+implementing metadata validation web services which are referenced
```

Comment:
Maybe you could add 'fleet' here, e.g. 'implementing the fleet of metadata validation web services'